scripts/ci/tags.yaml

```diff
@@ -477,3 +477,21 @@ cellular:
     - modules/lib/cjson/
     - modules/lib/hostap/
     - modules/lib/zcbor/
+
+app_event_manager:
+  files:
+    - modules/crypto/mbedtls/
```

Comment:
Event Manager by itself does not use a lot of these dependencies:

- `modules/crypto/mbedtls/`
- `modules/crypto/oberon-psa-crypto/`
- `nrf/drivers/hw_cc3xx/`
- `nrf/lib/modem_antenna/`
- `nrf/lib/nrf_modem_lib/`
- `nrf/subsys/nrf_security/`
- `nrfxlib/crypto/`
- `nrfxlib/nrf_modem/`
- `zephyr/soc/ti/lm3s6965/`
- `zephyr/subsys/ipc/`

Are you trying to cover dependencies of apps that use Event Manager (they seem not tagged now)? Maybe we could cover them with another tag to avoid confusion.

Reply:
On the other hand we may want to add dependency on Nordic boards from both `nrf` and `zephyr`. We could also consider adding dependency on Zephyr kernel features

Reply:
Discussed offline and aligned list of dependencies together